icu4c/source/common/common_uwp.vcxproj

```diff
@@ -106,7 +107,7 @@
       <TurnOffAssemblyGeneration>true</TurnOffAssemblyGeneration>
       <IgnoreSpecificDefaultLibraries>vccorlib.lib;msvcrt.lib</IgnoreSpecificDefaultLibraries>
       <!-- The icudt.lib is for U_ICUDATA_ENTRY_POINT -->
-      <AdditionalDependencies>icudt.lib;onecore.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Perhaps the [arm build](https://github.com/unicode-org/icu/actions/runs/11951328578/job/33314736962#step:5:4548) is failing due to this change ?